.travis.yml

```diff
@@ -6,12 +6,17 @@ cache:
   bundler: true
   directories:
   - tmp/cache/assets/test
+  - $HOME/.nvm
+  - $HOME/.yarn-cache
 node_js:
```

Comment:
Nopers, we had it in the main frontend and it was just tricking us the whole time